src/Microsoft.Azure.SignalRService/Credential.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.SignalRService
+{
+    public class Credential
+    {
+        public string Endpoint { get; set; }
+        public string AccessKey { get; set; }
+
+        private static readonly char[] PropertySeparator = { ';' };
+        private static readonly char[] KeyValueSeparator = { '=' };
+        private const string EndpointProperty = "endpoint";
+        private const string AccessKeyProperty = "accesskey";
+
+        public Credential()
+        {
+        }
+
+        internal void ParseConnectionString(string connectionString)
```

Comment:
I think our SDK already has some logic for parsing connection string?

Reply:
Our SDK make the logic internal. I consider making it public and use it instead.